projects/sandbox/src/app/sandbox-navigation/sandbox-navigation.component.ts

```diff
@@ -513,7 +513,16 @@ export class SandboxNavigatonComponent extends DataPollingComponent implements O
    *
    **/
   progressComplete(): boolean {
-    return !!this.progressData && this.progressData.status === DatasetStatus.COMPLETED;
+    const data = this.progressData;
+    if (data) {
+      if (data['dataset-logs'].length > 0) {
```

Comment:
Check for the status instead, there  is a new status value from backend `FAILED` when the dataset processing fails

Reply:
Done